include/eosio/vm/execution_context.hpp

```diff
@@ -302,6 +333,62 @@ namespace eosio { namespace vm {
          }
          __builtin_unreachable();
       }
+
+      int backtrace(void** out, int count, void* uc) const {
+         static_assert(EnableBacktrace);
+         void* end = this->_top_frame;
```

Comment:
Don't `end` & `rbp` need to be `volatile`?

Reply:
I don't think so.  I'm assuming that the signal handler is sequenced between instructions in the generated code.  If the signal happens during a host call, then it is unsequenced with respect to the host function, but that doesn't matter because `backtrace` only reads data that is written by the generated code or inline assembly.  If this assumption holds then `volatile` is unnecessary.  If it does not hold, then `volatile` won't help.